libbeat/registry/backend/memlog/testdata/1/logfile_only/meta.json

```diff
@@ -0,0 +1 @@
+{"version":1}
```

Comment:
mega minor: newline

Reply:
There should be no newline at the end. The diskstore just serializes to json, without adding any further whitespace.